mne/cov.py

```diff
@@ -1960,38 +1935,9 @@ def _estimate_rank_meeg_signals(data, info, scalings, tol='auto',
     return out
 
 
+@copy_doc(_estimate_rank_meeg_signals)
 def _estimate_rank_meeg_cov(data, info, scalings, tol='auto',
                             return_singular=False):
-    """Estimate rank for M/EEG data.
-
-    Parameters
-    ----------
-    data : np.ndarray of float, shape (n_channels, n_channels)
-        The M/EEG covariance.
-    info : Info
-        The measurment info.
-    scalings : dict | 'norm' | np.ndarray | None
-        The rescaling method to be applied. If dict, it will override the
-        following default dict:
-
-            dict(mag=1e12, grad=1e11, eeg=1e5)
-
-        If 'norm' data will be scaled by channel-wise norms. If array,
-        pre-specified norms will be used. If None, no scaling will be applied.
-    tol : float | str
-        Tolerance. See ``estimate_rank``.
-    return_singular : bool
-        If True, also return the singular values that were used
-        to determine the rank.
-
-    Returns
-    -------
-    rank : int
-        Estimated rank of the data.
-    s : array
-        If return_singular is True, the singular values that were
-        thresholded to determine the rank are also returned.
-    """
```

Comment:
Hmm... this one actually makes the code less readable/usable *during development*, i.e. when staring at code. I guess it's okay in this instance, but there are probably some cases where the evils of duplication would be worth it, e.g. in the complicated bits of `forward` code if we eventually have examples of duplication there.